app/views/two_factor_authentication/backup_code_verification/show.html.erb

```diff
@@ -12,7 +12,7 @@
       html: { autocomplete: 'off', method: :post },
     ) do |f| %>
   <%= render 'partials/backup_code/entry_fields', f: f, attribute_name: :backup_code %>
-  <%= f.submit t('forms.buttons.submit.default') %>
+  <%= f.submit t('forms.buttons.submit.default'), class: 'display-block margin-y-5' %>
```

Comment:
The changes here and the next two files are to maintain the vertical margin that was previously applied by `_fallback_links.html.erb`.